classes/Divisions.php

```diff
@@ -37,6 +37,24 @@ public function __construct(Member $member, PolicyPositions $positions = NULL, P
         $this->db = new \ParlDB;
     }
 
+    public static function getMostRecentDivisionDate() {
+      $db = new \ParlDB;
+      $q = $db->query(
+        "SELECT policy_id, max(division_date) as recent from policydivisions GROUP BY policy_id"
+      );
+
+      $policy_maxes = array();
+      $policy_maxes['latest'] = '';
+      $row_count = $q->rows();
+      for ($n = 0; $n < $row_count; $n++) {
+        $policy_maxes[$q->field($n, 'policy_id')] = $q->field( $n, 'recent' );
+        if ($q->field($n, 'recent') > $policy_maxes['latest']) {
+          $policy_maxes['latest'] = $q->field($n, 'recent');
+        }
+      }
```

Comment:
You could not have the `latest` bit inside and the loop and have `$policy_maxes['latest'] = max(array_values($policy_maxes));` after 🤷‍♂️